pkgs/applications/audio/soundconverter/default.nix

```diff
@@ -67,6 +67,11 @@ python3Packages.buildPythonApplication rec {
     runHook postCheck
   '';
 
+  postInstall = ''
+    wrapProgram $out/bin/soundconverter \
+      --prefix PYTHONPATH : "$PYTHONPATH:$out/lib:"
```

Comment:
```suggestion
      --prefix PYTHONPATH : "$out/lib"
```

Reply:
Fixed.

Reply:
I found out just doing:
```sh
wrapProgram $out/bin/soundconverter --prefix PYTHONPATH : "$PYTHONPATH"
```
Fixes this too.